packages/notifier/README.md

```diff
@@ -51,36 +62,33 @@ publication.updateState('a');
 publication.updateState('b');
 publication.finish('done');
 ```
-The publisher has published an interation with the non-final sequence
-`'a'`, `'b'`, terminated with `'done'` as its completion value.
 
-Alice the subscriber consumes using the `for-of-await` loop
+Alice, the subscriber, consumes the iteration using the `for-await-of` loop. She can see the 
+non-final values and whether the iteration completes or fails. She can see a failure reason, 
+but the `for-await-of` syntax does not let her see the completion value `'done'`. While she 
+can write code that only executes after the loop finishes, the code won’t know if the completion 
+value was “done”, “completed”, or something else. This is a limitation of JavaScript's iteration, 
+whether asynchronous or synchronous (as consumed by a `for-of` loop).
+
 ```js
 const consume = async subscription => {
   try {
     for await (const val of subscription) {
-      console.log('non-final', val);
+      console.log('non-final ', val);
     }
-    console.log('finished');
+    console.log('the iteration finished');
   } catch (reason) {
-    console.log('failed', reason);
+    console.log('the iteration failed ', reason);
```

Comment:
Kill extra space on the end. `console.log` does that for you. If you do it too, you'll get two spaces.
```suggestion
    console.log('the iteration failed', reason);
```